scripts/devaccess/devaccess_wrap.py

```diff
@@ -289,6 +289,19 @@ def oc_get_pods(occmd):
 
         return results
 
+    @staticmethod
+    def oc_get_routes(occmd):
+        ''' provide 'oc get routes' '''
+
+        n_cmd = occmd.normalized_cmd()
+        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
+        # -o yaml and -o json output of routes can expose
```

Comment:
@dbaker-rh we don't currently whitelist 'oc get routes -o yaml|json', but if we did, this would be the output:

# oc get routes -n default docker-registry -o yaml
apiVersion: v1
kind: Route
metadata:
  creationTimestamp: 2017-07-06T19:35:24Z
  name: docker-registry
  namespace: default
  resourceVersion: "2042"
  selfLink: /oapi/v1/namespaces/default/routes/docker-registry
  uid: 41dd80e1-6282-11e7-be32-0a73df9a15cc
spec:
  host: <host name the route is servicing here>
  tls:
    caCertificate: |
      -----BEGIN CERTIFICATE-----
      public CA certificate here
      -----END CERTIFICATE-----
    certificate: "public-facing certificate here"
    destinationCACertificate: |
      -----BEGIN CERTIFICATE-----
      internal cluster CA certificate here
      -----END CERTIFICATE-----
    key: |
      -----BEGIN PRIVATE KEY-----
      PRIVATE KEY DATA HERE!!!          <----- houston...we have a problem
      -----END PRIVATE KEY-----
    termination: reencrypt
  to:
    kind: Service
    name: docker-registry
    weight: 100
  wildcardPolicy: None

We'd need to remove/redact .spec.tls.key